std_msgs/Header header

# the waypoint id
string id

# the snapshot id
string snapshot_id

# pose of waypoitn in seed frame
geometry_msgs/Pose pose_sf

# annotation name
string name